dialogues/risawoz/src/convert.py

```diff
@@ -340,7 +345,7 @@ def build_dataset(original_data_path, db, setting, debug=False):
                     if "MONGODB_HOST" in os.environ
                     else 'mongodb+srv://bitod:plGYPp44hASzGbmm@cluster0.vo7pq.mongodb.net/risawoz?retryWrites=true&w=majority&ssl=true&ssl_cert_reqs=CERT_NONE')
 
-    if build_db:
+    if not args.no_build_db:
```

Comment:
In `convert.py`, we only use local database. This is 1) to ensure remote database is not modified 2) querying local database is much faster than remote database. There's about 10X difference in processing speed.

The remote database is only used during evaluation at inference time.